addons/l10n_at/data/account_account_tag.xml

```diff
@@ -7,370 +7,423 @@
         </record>
         <record
             id="account_tag_l10n_at_ABI4"
             model="account.account.tag">
             <field name="name">Bilanz ABI4</field>
             <field name="applicability">accounts</field>
             <field name="color" eval="1" />
+            <field name="country_id" ref="base.at"/>
         </record>
         <record
             id="account_tag_l10n_at_ABI5"
             model="account.account.tag">
             <field name="name">Bilanz ABI5</field>
             <field name="applicability">accounts</field>
             <field name="color" eval="1" />
+            <field name="country_id" ref="base.at"/>
         </record>
         <record
             id="account_tag_l10n_at_ABII1"
             model="account.account.tag">
             <field name="name">Bilanz ABII1</field>
             <field name="applicability">accounts</field>
             <field name="color" eval="1" />
+            <field name="country_id" ref="base.at"/>
         </record>
         <record
             id="account_tag_l10n_at_ABII2"
             model="account.account.tag">
             <field name="name">Bilanz ABII2</field>
             <field name="applicability">accounts</field>
             <field name="color" eval="1" />
+            <field name="country_id" ref="base.at"/>
         </record>
         <record
             id="account_tag_l10n_at_ABII3"
             model="account.account.tag">
             <field name="name">Bilanz ABII3</field>
             <field name="applicability">accounts</field>
             <field name="color" eval="1" />
+            <field name="country_id" ref="base.at"/>
         </record>
         <record
             id="account_tag_l10n_at_ABII4"
             model="account.account.tag">
             <field name="name">Bilanz ABII4</field>
             <field name="applicability">accounts</field>
             <field name="color" eval="1" />
+            <field name="country_id" ref="base.at"/>
         </record>
         <record
             id="account_tag_l10n_at_ABIII"
             model="account.account.tag">
             <field name="name">Bilanz ABIII</field>
             <field name="applicability">accounts</field>
             <field name="color" eval="1" />
+            <field name="country_id" ref="base.at"/>
         </record>
         <record
             id="account_tag_l10n_at_ABIII1"
             model="account.account.tag">
             <field name="name">Bilanz ABIII1</field>
             <field name="applicability">accounts</field>
             <field name="color" eval="1" />
+            <field name="country_id" ref="base.at"/>
         </record>
         <record
             id="account_tag_l10n_at_ABIII2"
             model="account.account.tag">
             <field name="name">Bilanz ABIII2</field>
             <field name="applicability">accounts</field>
             <field name="color" eval="1" />
+            <field name="country_id" ref="base.at"/>
```

Comment:
I am not sure this is necessary when they are of type account.  (main question would be if it helps to put it because we do not even filter on it in the views...)

Reply:
It was a PO suggestion. It is visible / usable (e.g. filter) in the account tags list view (when searching for tags ("Search More...") when assigning them in an account form view or in the list view in the configuration menu (debug mode only I guess)). I do not think it hurts to have the information? But I can remove it if is a problem.

Reply:
Ok, but then we might go all the way...